Makefile

```diff
@@ -42,6 +43,9 @@ initialrelease: python
 test: python
 	@tox -qe tests,py27-tests
 
+.PHONY: sure
+sure: checkformatting lint test
```

Comment:
We should add `make sure` to the cookiecutter (https://github.com/hypothesis/h-cookiecutter-pypackage/), then we can update all of our Python packages with it

Reply:
Done :+1: